### DISCUSSION TITLE
Playbook Community Sigma Rules missing?

### DISCUSSION THREAD

**Discussion by mtnsec** [Security-Onion-Solutions/securityonion#3713] [Unsupported Versions] [answered]
I'm just now starting to dig into Playbook, and after reading the documentation, I think I might have some issues?

When I click the Playbook link from the SOC interface it opens a new window to Playbook, but there are zero plays. I've checked all of the query links, and the total number of plays is zero. The docs indicate that there should be 500+ preloaded plays. How do I find them?

Thanks!! :)

**Comment by presianbg** [comment, marked as answer]
Try to run  `sudo so-playbook-ruleupdate` on your manager node and investigate the output.

**Reply by mtnsec** [reply]
Looks like I have two types of errors.  

First I have hundreds of these:
```
Play Creation Error - 
Play Creation Error - 
Play Creation Error - 
Play Creation Error - 
Play Creation Error - 
Play Creation Error - 
```

And a hundreds of these (the error is the same for every .yml)
```
sigma/rules/windows/process_creation/win_data_compressed_with_rar.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}
new
```

**Reply by defensivedepth** [reply]
hmmm. Go ahead and do a full reset on Playbook:  `sudo so-playbook-reset`

This will take a while to complete.

**Reply by mtnsec** [reply]
Thanks that seems to have fixed it!

Looks like all of the community rules are for windows events? Do you have any resources for non-windows (linux, zeek/network, etc.) examples that I can use to learn more?  I don't have windows on my network.

**Reply by presianbg** [reply]
Hi, 
Take a look at the [docs](https://docs.securityonion.net/en/2.3/playbook.html?highlight=playbook#adding-additional-rulesets):
![image](https://user-images.githubusercontent.com/2163486/113423673-8e532f00-93d7-11eb-8e8f-f26b1064eda6.png)


I would recommend SOC Prime marketplace for more sigma rules :)

**Reply by mtnsec** [reply]
Thanks!!

**Reply by defensivedepth** [reply]
Great to hear @mtnsec!

Thanks for the assist @presianbg !
